tsconfig.json

```diff
@@ -20,5 +20,5 @@
         }
     },
     "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
-    "exclude": ["node_modules"]
+    "exclude": ["node_modules", "components/common/ClientOnlyPortal.ts"]
```

Comment:
What is this file and why is it excluded?

Reply:
That file doesn't exist (anymore). It was never pushed to github, but I forgot to remove that exclusion. Fixed in 9137767